core/jobs_registry.py

```diff
@@ -226,6 +226,10 @@ class ContinuousComputationEventDispatcher(python_utils.OBJECT):
     def dispatch_event(cls, event_type, *args, **kwargs):
         """Dispatches an incoming event to the ContinuousComputation
         classes which listen to events of that type.
+
+        Args : str . Type of event.
```

Comment:
This is wrong. What's the argument name?

Reply:
So should I just specify the argument name as "event_type" ?

Reply:
``@classmethod
    def dispatch_event(cls, event_type, *args, **kwargs):
        """Dispatches an incoming event to the ContinuousComputation
        classes which listen to events of that type.

        Args :
            event_id: str . Type of event.
        """